app/routes/candidates.js

```diff
@@ -3,9 +3,10 @@ var router = express.Router();
 var sqlite3 = require('sqlite3').verbose();
 var db = new sqlite3.Database('../db/db.sqlite3');
 
-/* GET users listing. */
+/* GET candidates listing. */
 router.get('/', function(req, res, next) {
   var candidates = [];
+  var selectClauseGenElect = 'SELECT coalesce(max(gen_election_candidate), \'0\') AS generalElection FROM candidates';
```

Comment:
what does this mean? is it checking if there is any candidate with non null value of 'gen_election_candidate'?